Library/Formula/docbook.rb

```diff
@@ -43,9 +43,6 @@ class Docbook < Formula
   end
 
   def install
-    (etc/"xml").mkpath
-    system "xmlcatalog", "--noout", "--create", "#{etc}/xml/catalog"
```

Comment:
If I would do that, wouldn't I still have the sandbox problem as I am creating a file outside of the keg during `install`?